app/src/main/java/org/mozilla/focus/browser/HomeTileRecyclerView.kt

```diff
@@ -114,6 +118,13 @@ class HomeTileAdapter(
         notifyItemInserted(homeTiles.lastIndex)
     }
 
+    fun getItemAtPosition(position: Int): HomeTile? {
```

Comment:
nit: You can shorten this a tad and make it more readable:
```kotlin
fun getItemAtPosition(position: Int): HomeTile? = if (position > -1 && position < itemCount) {
    tiles[position]
} else {
    null
}
```

It may seem like a trivial difference but I find 1) keeping the file short can improve readability and 2) having the indented section display the return value makes it clear where the checks are and where the return values are.